css/components/heading.css

```diff
@@ -56,10 +60,8 @@ p.section-description.about-me {
 }
 
 .section-headline {
-  font-weight: 400;
```

Comment:
be reikalo ištrynei, susigadino hero ir about me sekcijos.

Reply:
įrašiau atgal